docs/prepare-a-shared-rp-development-environment.md

```diff
@@ -301,7 +314,8 @@ locations.
    ADMIN_OBJECT_ID='$ADMIN_OBJECT_ID'
    PARENT_DOMAIN_NAME='$PARENT_DOMAIN_NAME'
    PARENT_DOMAIN_RESOURCEGROUP='$PARENT_DOMAIN_RESOURCEGROUP'
-   export DOMAIN_NAME="\$RESOURCEGROUP.\$PARENT_DOMAIN_NAME"
+   export DOMAIN_NAME="\$LOCATION.\$PARENT_DOMAIN_NAME"
```

Comment:
in the env file that is on my local machine, we are not escaping the $.

`export DOMAIN_NAME="$LOCATION.$PARENT_DOMAIN_NAME"`

Reply:
@mikeandescavage the backslash in this `cat` command ensures that it ends up the way you have it above; otherwise, it would put the current value of `LOCATION` there instead.

Reply:
@nilsanderselde  yep... this is what I get for comparing the output.  This is a non-issue.